src/rules.rs

```diff
@@ -55,7 +55,6 @@ pub mod no_invalid_regexp;
 pub mod no_invalid_triple_slash_reference;
 pub mod no_irregular_whitespace;
 pub mod no_misused_new;
-pub mod no_mixed_spaces_and_tabs;
```

Comment:
Should this rule exist? I think it's a formatting concern of `deno fmt` and not `deno lint` so I removed it, but I mainly removed it in this PR because it would be a bit of work to update it to work now (because it was using `source_map().span_to_lines(...)`).